src/util/prometheus.js

```diff
@@ -633,6 +620,56 @@ class Prometheus {
         type: 'histogram',
         labelNames: [],
       },
+      {
+        name: 'http_request_size',
+        help: 'http_request_size',
+        type: 'histogram',
+        labelNames: ['method', 'route', 'code'],
+        buckets: [
+          1000, 10000, 100000, 500000, 1000000, 1500000, 2000000, 2500000, 3000000, 3500000,
+          4000000, 4500000, 5000000, 10000000, 15000000, 20000000, 25000000, 30000000, 35000000,
+          40000000, 45000000, 50000000,
+        ],
+      },
+      {
+        name: 'http_response_size',
+        help: 'http_response_size',
+        type: 'histogram',
+        labelNames: ['method', 'route', 'code'],
+        buckets: [
+          1000, 10000, 100000, 500000, 1000000, 1500000, 2000000, 2500000, 3000000, 3500000,
+          4000000, 4500000, 5000000, 10000000, 15000000, 20000000, 25000000, 30000000, 35000000,
+          40000000, 45000000, 50000000,
+        ],
+      },
+      {
+        name: 'dest_transform_input_events',
+        help: 'dest_transform_input_events',
+        type: 'histogram',
```

Comment:
Explain: Changed the type of these metrics from counter to histogram. This is done to observe the distribution of the events length in requests.